Readme.md

```diff
@@ -109,51 +186,69 @@ if (true)
 }
 ```
 
-Also, notice the use of whitespace before and after the condition statement.
-
-## Method chaining
-
-One method per line should be used if you want to chain methods.
+#### Newline character after open braces
 
-You should also indent these methods so it's easier to tell they are part of the same chain.
+Always insert a line break following the open brace `{` character, except for
+object literals or immediately closed brace `}` character.
 
 *Right:*
 
 ```js
-User
-  .findOne({ name: 'foo' })
-  .populate('bar')
-  .exec(function(err, user) {
-    return true;
-  });
-````
+function emptyFunc() {}
+
+function trivialFunc() {
+  return false;
+}
+
+if (trivialBoolean) {
+  return false;
+} else {
+  doSomething();
+}
+
+var obj = {'a': 1, 'b': 2};
+```
 
 *Wrong:*
 
 ```js
-User
-.findOne({ name: 'foo' })
-.populate('bar')
-.exec(function(err, user) {
-  return true;
-});
+function trivialFunc() { return false; }
 
-User.findOne({ name: 'foo' })
-  .populate('bar')
-  .exec(function(err, user) {
-    return true;
-  });
+if (trivialBoolean) {  return false; }
+```
 
-User.findOne({ name: 'foo' }).populate('bar')
-.exec(function(err, user) {
-  return true;
-});
+### Alphabetize dependencies
```

Comment:
Not about alphabetizing dependencies, but here are some proposed rules about dependencies that have come up for me as I've dealt with this codebase:

(1) i'd like a rule that says requires should be declared with separate `var`s (or one per statement basically). the alternative is harder to deal with when adding dependencies later.
(2) I'd also like us to _not_ indent the `=` (line them up) in require statements since it's also harder to deal with when later adding dependencies.